test/e2e/addon/podidentityverify.go

```diff
@@ -27,19 +28,27 @@ type VerifyPodIdentityAddon struct {
 func (v VerifyPodIdentityAddon) Run(ctx context.Context) error {
 	v.Logger.Info("Verify pod identity add-on is installed")
 
-	podIdentityAddon := Addon{
-		Name:    podIdentityAgent,
-		Cluster: v.Cluster,
+	podIdentityAddon := PodIdentityAddon{
```

Comment:
can this just be a `Addon` instead?
it seems like we don't need the pod identity struct since we only want to wait until it's active?